docs/user_guide/configuration.rst

```diff
@@ -0,0 +1,124 @@
+Guide to ``btrack`` configuration parameters
+============================================
+
+This is a short guide to the configuration parameters for ``btrack``
+
+Miscellaneous parameters
+------------------------
+
+-  ``max_search_radius`` - the maximum search radius in isotropic unit
+   of the data
+-  ``mode`` - the update mode for the tracker
+-  ``volume`` - estimate of the dimensions of the imaging volume
+
+Motion model
+------------
+
+.. code:: json
+
+   "MotionModel": {
+     "name": "cell_motion",
+     "dt": 1.0,
+     "measurements": 3,
+     "states": 6,
+     "accuracy": 7.5,
+     "prob_not_assign": 0.001,
+     "max_lost": 5,
+     "A": {
+       "matrix": [1,0,0,1,0,0,
+                  0,1,0,0,1,0,
+                  0,0,1,0,0,1,
+                  0,0,0,1,0,0,
+                  0,0,0,0,1,0,
+                  0,0,0,0,0,1]
+     },
+     "H": {
+       "matrix": [1,0,0,0,0,0,
+                  0,1,0,0,0,0,
+                  0,0,1,0,0,0]
+     },
+     "P": {
+       "sigma": 150.0,
+       "matrix": [0.1,0,0,0,0,0,
+                  0,0.1,0,0,0,0,
+                  0,0,0.1,0,0,0,
+                  0,0,0,1,0,0,
+                  0,0,0,0,1,0,
+                  0,0,0,0,0,1]
```

Comment:
is this matrix correct? I see `0.1` in some entries but `1` elsewhere?

Reply:
Issue for another PR.

Reply:
https://github.com/quantumjot/BayesianTracker/issues/65